### DISCUSSION TITLE
Option to allow beacons to work without exposed sky.

### DISCUSSION THREAD

**Discussion by mibby** [PurpurMC/Purpur#830] [Feature Requests]
I have a custom nether that generates terrain from Y 0 to Y 255 with the Amplified-Nether datapack. Players have been asking for a way to allow beacons to work in the nether since they cannot build on the nether roof anymore. Beacons only work when they have an exposed sky. I cannot find a way to increase nether build height past 255. Thus, this request.

Would it be possible to add an option to allow beacons to work without being exposed to the sky unobstructed?

**Comment by BillyGalbreath** [comment]
Yes and no. I could make the change on the server, sure. It would apply the potion effects as expected, but there will not be any visual beam coming out of the beacon. The beam is controlled client side.

**Reply by mibby** [reply]
I don't mind the visual effect not being there if it's clientside.
